core-js-part2/task1.js

```diff
@@ -0,0 +1,17 @@
+function getSumOfEvenArrayNumbers(array) {
+  let sum = 0;
+  for (item of array) {
+    if (typeof item === "number" && item % 2 === 0) {
```

Comment:
In front-end there are many options to have a number as a string type (inputs, server response, etc.). It would be better to convert the value to a numeric type with Number().

Number will return a value of type "number" if the value provided is a number or a string including only digits. If you provide a string with letters or any other data types, Number will return "NaN".

References:
Number: https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Number
NaN: https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/NaN

Reply:
Fixed!